### PR TITLE
Feat(eos_cli_config_gen): Add schema for ipv6_hardware

### PR DESCRIPTION
**Pull request by KonikaChaurasiya-GSLab**
### Contributor Checklist

- [x] Create schema fragment matching data model described in README.md and README_v4.0.md
  - README.md is most complete with all keys. README_v4.0 includes partial data models after conversion to lists.
  - Schema fragment path is `roles/eos_cli_config_gen/schemas/schema_fragments/<data_model_key>.schema.yml`.
  - Copy line 1-5 from another schema (comments and outer type:dict).
  - Refer to [schema documentation](https://avd.sh/en/devel/docs/input-variable-validation-BETA.html) for syntax and/or use YAML Lint plugin from Redhat in VSCode.
  - Use `convert_types` on value that could be mixed type or misinterpreted like integers and numeric strings.
- [x] If the data model has been converted from wildcard dicts:
  - Add `convert_types: ['dict']` to the schema.
  - Remove `convert_dicts` from the `templates/eos/<>.j2` and `templates/documentation/<>.j2` templates.
- [x] Run `molecule converge -s build_schemas_and_docs` to update schema and documentation.
- [x] Test by running `molecule converge -s eos_cli_config_gen` and verify no errors or changes to generated configs/docs.

### Reviewer Checklist

- Reviewer 1:
  - [x] Verify that data model is fully covered in the described schema. Easiest by looking at the generated documentation.
  - [x] Verify that `convert_dicts` has been removed from templates as applicable.
  - [x] Verify no changes to configs/docs on any molecule scenario
  - [x] Verify that CI pass

- Reviewer 2:
  - [x] Verify that data model is fully covered in the described schema. Easiest by looking at the generated documentation.
  - [x] Verify that `convert_dicts` has been removed from templates as applicable.
  - [x] Verify no changes to configs/docs on any molecule scenario
  - [x] Verify that CI pass

### CODE REVIEW

**Review by tgodaA — APPROVED** [review]
LGTM

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/ipv6_hardware.schema.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        keys:
+          optimize:
+            type: dict
+            keys:
+              prefixes:
+                type: dict
+                keys:
+                  profile:
+                    type: str
+                    valid_values: [ "internet"]
```
Comment (carlbuchmann) [inline ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/ipv6_hardware.schema.yml]: There could be other valid values if a user creates a custom profile, so we would need to remove this. 
I would recommend adding a description stating:
"pre-defined profile "internet" or user-defined profile.

Reference:

"Once the TCAM profile is applied, you can enable FlexRoute compression and expansion of prefixes.  While these commands can be customized for different prefix lengths, Arista has included pre-defined 'internet' profiles for both IPv4 and IPv6 based on the current contents of the Internet routing table."

https://aristanetworks.force.com/AristaCommunity/s/article/inet-edge-config

**Review by carlbuchmann — DISMISSED** [review]
We need to remove `valid_values` on `ipv6_hardware.fib.optimize.prefixes.profile`. Please see the comment.

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/ipv6_hardware.schema.yml (hunk 1 above)
Comment (ClausHolbechArista) [reply ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/ipv6_hardware.schema.yml]: done

**Review by ClausHolbechArista — APPROVED** [review]
LGTM

**Pull request merged**